packages/pwa-kit-extension-sdk/src/configs/babel/utils.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fse from 'fs-extra'
+import * as p from 'path'
+import {getApplicationExtensionInfo} from '../../shared/utils/extensibility'
+
+/**
+ * Builds Babel extensibility arguments for processing specific files and paths.
+ * Accessing files via "node_modules" ensures compatibility in both mono-repos and
+ * developers' environments working on templates with extensibility. Avoids relative paths
+ * or symlinked paths that Babel doesn't support by using realpathSync.
+ */
+export const buildBabelExtensibilityArgs = () => {
+    const {installed}: {installed: (string | undefined)[]} = getApplicationExtensionInfo()
+
+    // Filter out undefined values from installed extensions
+    const filteredInstalled = installed.filter(
+        (packageName): packageName is string => packageName !== undefined
+    )
+
+    const serverPath = fse.realpathSync(
+        p.resolve('node_modules/@salesforce/pwa-kit-runtime/ssr/server/build-remote-server.js')
```

Comment:
Lets make constants for these strings, or in the unlikely scenario we do ever change the path of those src files, we'll have an easier time spotting those constants and updating them.